pkg/cmd/workspace/util/creation_data.go

```diff
@@ -29,16 +29,25 @@ func GetCreationDataFromPrompt(config CreateDataPromptConfig) (string, []apiclie
 	var providerRepo *apiclient.GitRepository
 	var err error
 	var workspaceName string
+	// Below 3 maps keeps track of visited repos, their respective namespaces and git providers.
```

Comment:
Change to
"// The following three maps keep track of visited repos and their respective namespaces and Git providers"

Reply:
fixed.